themes/digital.gov/layouts/partials/core/read-time.html

```diff
@@ -1,21 +1,19 @@
 {{- $guide_data := index $.Site.Data.guidenav (.Params.guide) -}}
+{{- $readTime := 0 -}}
 
-{{- if and $guide_data $guide_data.showReadTime -}}
-
-  {{- $readTime := 0 -}}
+{{- /* Show read time by default. Only disable read time if parameter is set explicitly in guidenav file */ -}}
 
+{{- if or (not (and $guide_data (eq $guide_data.showReadTime false))) (not $guide_data) -}}
```

Comment:
We could improve clarity of this `if` statement by extracting the check into it's own variable, like `showReadTime`

Reply:
Thanks! Addressed in [68462d2](https://github.com/GSA/digitalgov.gov/pull/6316/commits/68462d2fbd47c3a8d26fb81f36c423f6698268c0)